src/main/io/vtx_smartaudio.c

```diff
@@ -1022,6 +1028,20 @@ static bool vtxSAGetStatus(const vtxDevice_t *vtxDevice, unsigned *status)
     return true;
 }
 
+static uint8_t vtxSAGetPowerLevels(const vtxDevice_t *vtxDevice, uint16_t *levels)
+{
+    if (!vtxSAIsReady(vtxDevice) || saDevice.version < 2) {
+        return 0;
+    }
+
+    for (uint8_t i = 0; i < saSupportedNumPowerLevels; i++)
+    {
```

Comment:
Opening brace goes onto the same line except for function definitions.

Reply:
Fixed